src/components/HQCalendar.tsx

```
@@ -0,0 +1,882 @@
+        case 'm':
+          setViewMode('monthly');
+          break;
+        case 'y':
+          setViewMode('year');
+          break;
+        case 's':
+          setViewMode('schedule');
+          break;
+        case 't':
+          // Go to today - inline to avoid dependency issues
+          const today = new Date();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)